types/types.go

```diff
@@ -52,7 +55,7 @@ const (
 	// 3. Maximum kubernetes name length is 63
 	// 4. cronjob pod suffix is 11
 	// 5. Dash and buffer for 2
-	MaximumJobNameSize = 8
+	MaximumJobNameSize = 30
```

Comment:
I'm not pretty sure how to calculate this const variable. When I'm testing `Recurring backup and snapshot` from UI, it returns an error because this value is too short for a CronJob name. So I change it to a random value this time. 
I would be appreciate for your help.

Reply:
Use new version of UI: rancher/longhorn-ui:6d74dfc , see https://github.com/rancher/longhorn-manager/commit/1e5f85cfe8cc02112a690510b51564697fd18f14

Reply:
And separate this to another commit next time.